sebs/aws/aws.py

```
@@ -204,42 +229,53 @@ def create_function(self, code_package: Benchmark, func_name: str) -> "LambdaFun
-                code_bucket = storage_client.get_bucket(Resources.StorageBucketType.DEPLOYMENT)
-                code_prefix = os.path.join(benchmark, architecture, code_package_name)
-                storage_client.upload(code_bucket, package, code_prefix)
-
-                self.logging.info("Uploading function {} code to {}".format(func_name, code_bucket))
-                code_config = {"S3Bucket": code_bucket, "S3Key": code_prefix}
-            ret = self.client.create_function(
-                FunctionName=func_name,
-                Runtime="{}{}".format(
+                create_function_params["PackageType"] = "Zip"
+                if code_size < 50 * 1024 * 1024:
+                    package_body = open(package, "rb").read()
```

Ruff

Use a context manager for opening files

(SIM115)